tests/test_detectAndMeasure.py

```diff
@@ -217,6 +265,261 @@ def _check_diaSource(self, refSources, diaSource, refIds=None,
                                          rtol=rtol, atol=atol)
 
 
+class DetectAndMeasureScoreTest(lsst.utils.tests.TestCase):
+
+    def test_detection_runs(self):
+        """Basic smoke test.
+        """
+        noiseLevel = 1.
+        staticSeed = 1
+        fluxLevel = 500
+        kwargs = {"seed": staticSeed, "psfSize": 2.4, "fluxLevel": fluxLevel, "x0": 0, "y0": 0}
+        science, sources = makeTestImage(noiseLevel=noiseLevel, noiseSeed=6, **kwargs)
+        matchedTemplate, _ = makeTestImage(noiseLevel=noiseLevel/4, noiseSeed=7, **kwargs)
+        difference = science.clone()
+        subtractTask = subtractImages.AlardLuptonPreconvolveSubtractTask()
+        scienceKernel = science.psf.getKernel()
+        score = subtractTask._convolveExposure(difference, scienceKernel, subtractTask.convolutionControl)
+        config = detectAndMeasure.DetectAndMeasureScoreTask.ConfigClass()
+        config.doApCorr = False
+        task = detectAndMeasure.DetectAndMeasureScoreTask(config=config)
+        output = task.run(science, matchedTemplate, difference, score)
+        subtractedMeasuredExposure = output.subtractedMeasuredExposure
+        self.assertImagesEqual(subtractedMeasuredExposure.image, difference.image)
+
+    def test_detection_xy0(self):
+        """Basic smoke test with non-zero x0 and y0.
```

Comment:
Reworded slightly. I find this test helpful since it has all optional processing turned off, so it helps identify where the problem is when there is one.